### PR TITLE
Re-add ios_signing dependency to ios_content_validation_test

### PR DESCRIPTION
**Pull request by jmagman**
https://github.com/flutter/infra/pull/287 moved ios_content_validation_test from devicelab to luci builders.  It failed on a try build with `No valid code signing certificates were found`.  Re-add `ios_signing` dependency removed in that PR.

https://github.com/flutter/infra/pull/287#discussion_r539705796

### CODE REVIEW

**Comment by jmagman** [comment]
@keyonghan I know this dependency was introduced for the devicelab staging in #262.  Will it also work in this config?

**Comment by keyonghan** [comment]
> @keyonghan I know this dependency was introduced for the devicelab staging in #262. Will it also work in this config?

Does this test need a phone device? We do not have any bot with a device available in the prod pool at this moment.

**Comment by jmagman** [comment]
> Does this test need a phone device? We do not have any bot with a device available in the prod pool at this moment.

No, it doesn't need a device, that's why I moved it in https://github.com/flutter/infra/pull/287.  It's archiving the app, which requires a provisioning profile.

File: config/devicelab_config.star [hunk 1]
```diff
@@ -541,6 +541,9 @@ def devicelab_prod_config(branch, version, ref):
                 {
                     "dependency": "gems",
                 },
+                {
```
Comment (godofredoc) [inline config/devicelab_config.star:544]: This dependency is only available in staging and not available yet for prod. The reason is signing will be only supported on devicelab_machines where we control the provisioning.

Comment (jmagman) [reply config/devicelab_config.star:544]: Filed https://github.com/flutter/flutter/issues/72078

**Comment by jmagman** [comment]
Closing, looks like this test can't be moved from `devicelab_machines`.

**Pull request closed without merging**